webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -64,7 +85,18 @@ export default class PostMessageView extends React.Component {
         return (
             <span
                 onClick={Utils.handleFormattedTextClick}
-                dangerouslySetInnerHTML={{__html: TextFormatting.formatText(this.props.message, options)}}
+                dangerouslySetInnerHTML={{
```

Comment:
Can you change this not to use dangerouslySetInnerHTML? We need that for markdown and text formatting, but we shouldn't need it for the (edited) label. You can probably  just wrap this whole thing in a div if necessary like
```
return (
    <div>
        <span dangerouslySetInnerHTML=.../>
        <span className='edited'>...</span>
    </div>
);
````